cmd/applications-rp/radius-cloud.yaml

```diff
@@ -1,56 +0,0 @@
-# This is an example of configuration file.
-environment:
-  name: AzureCloud
-  roleLocation: West US
-identity: # 1P AAD APP authentication
-  clientId: "PLACEHOLDER"
-  instance: "https://login.windows.net"
-  tenantId: "common"
-  armEndpoint: "https://management.azure.com:443"
-  audience: "https://management.core.windows.net"
-  pemCertPath: "/var/certs/rp-aad-app.pem"
-storageProvider:
-  provider: "cosmosdb"
-  cosmosdb:
-    # Create your own SQL API Cosmos DB account and set url in this configuration or to RADIUS_STORAGEPROVIDER_COSMOSDB_URL environment variable
-    url: https://radius-eastus-test.documents.azure.com:443/
```

Comment:
I thought it best to just remove this file. There's a bunch of Azure-specific stuff in here that's not relevant. I can't think of a scenario where this is used.